uptane-whitepaper.md

```diff
@@ -71,9 +71,9 @@ The second change made to the basic TUF design has to do with the way Uptane ver
 ![Uptane Process](https://github.com/uptane/papers/blob/master/images/Uptane_process.png)
 
 
-The diagram above illustrates how the checks and balances of the Uptane system works. The connected components on the right hand side of the diagram are on the vehicle, while the components on the left hand-side represent the repositories. The Image repository can be thought of as an authoritative source of information about software images, and the keeper of every image currently deployed by the OEM or supplier, along with the metadata files that prove their authenticity. The Director repository determines what software updates should be distributed to each ECU. 
+The diagram above illustrates how the checks and balances of the Uptane system works. The connected components on the right hand side of the diagram are on the vehicle, while the components on the left hand side represent the repositories. The Image repository can be thought of as an authoritative source of information about software images, and the keeper of every image currently deployed by the OEM or supplier, along with the metadata files that prove their authenticity. The Director repository determines what software updates should be distributed to each ECU. 
```

Comment:
this hunk just makes hand side vs. hand-side more consistent.

Reply:
fine.